packages/@n8n/ai-workflow-builder.ee/src/chains/prompts/examples/basic/switch-node-examples.ts

```diff
@@ -0,0 +1,124 @@
+export const SWITCH_NODE_EXAMPLES = `
```

Comment:
Love this - nice to start expanding examples like this!

One thing that comes to mind - it could be worth having a `README.md` to describe when to use node-types and when to use examples for these prompts, especially for agents working in this area, explanation of the difference could be helpful (not necessarily for this PR, just suggesting to get the idea out there!)